.gitignore

```diff
@@ -24,7 +24,8 @@ Makefile.in.in
 autom4te.cache
 libtool
 stamp-h1
-m4/
+m4/*
+!m4/ld-version-script.m4
```

Comment:
Could you store that file in different directory? Then we needn't change .gitingnore

e.g. `external//ld-version-script.m4`

Reply:
If we did that, we'd also have to change/add to the `AC_CONFIG_MACRO_DIR([m4])` in configure.ac. It's better to keep m4 scripts in the m4 directory.